ansible/group_vars/all

```diff
@@ -138,6 +138,8 @@ kafka:
   offsetsTopicReplicationFactor: "{{ kafka_offsetsTopicReplicationFactor | default(kafka_replicationFactor) | default((groups['kafkas']|length)|int) }}"
   # adapt this param for production deployments depending on the number of kafka consumers
   networkThreads: "{{ kafka_network_threads | default(3) }}"
+  payload:
+    max: "{{ limit_activation_payload + (limit_activation_payload * 0.05) | int | default() }}"
```

Comment:
what is the `0.05` constant meant to do?  provide some "buffer"?  if so, how was this value determined?